docs/non-normative/how-to-write-conventions/resource-and-entities.md

```diff
@@ -0,0 +1,285 @@
+  metric. There is *no* requirement on disjointness due to how is-a
+  relationships are modelled.
+
+### Extending an entity
+
+While not recommended for Semantic Conventions, you can define a new
+"view" of an entity that includes additional descriptive attributes. To
+do so, use the extends field on groups:
+
+`model/{my_other_domain}/entities.yaml`:
+
+```yaml
+groups:
+  - id: entity.my_entity_2
+    type: entity
+    extends: entity.my_entity
+    attributes:
+      - ref: new.attribute.name
+        requirement_level: opt_in
+        role: descriptive
+```
+
+Notes:
+
+- You cannot change the `name` or `type` fields of the new entity.
+- You cannot change the set of identifying attributes.
+
+## FAQ
+
+### When to define a new entity?
+
+There are two scenarios where entities should be defined:
+
+- When you are generating a new signal (log, metric, span, etc.)
+  and no existing entity makes sense as the "source".
+- (future) When you need to describe an entity hierarchy from
+  some system of record (e.g. resources in kubernetes, assets
+  in a cloud).
+
+For example, the Mainframe SIG in OpenTelemetry will need to outline a
+set of entities and relationships to define observed telemetry. If a new
+clustering solution (e.g. Hashicorp's Nomad) is defined, and existing
+container-based entities are not enough, then new entities should be
+defined.
+
+### When to define an "is-a" relationship vs. extending descriptive attributes?
+
+There are two key rules:
+
+- Default to introducing separate entities with a clear "is a" (or similar) relationship
+- Extend an entity with new descriptive attributes if and only if the following is true:
+  - The extending entity cannot be associated with any telemetry by itself.
+    *For example, when adding a `windows.process` entity, but no new
+    signals (metrics, logs, spans, etc.) would be added to the entity.*
+  - The extending entity doesn’t have any other entities that logically can
+    only be associated with it.
+    *Note: this only applies to entities as signals with relationships.*
+
+This avoids the complexities of subtyping and ambiguous attribute usage.
+
+### How to define identifying attributes?
+
+The identifying attributes should be minimally sufficient to
+identify an entity within the context of how that entity is
+discovered. For example, when discovering kubernetes entities
+like `k8s.pod`, `k8s.deployment`, the identifying attributes
+should be sufficient to identify these entities within the
+scope of a kubernetes cluster (or more specifically, the
+kubernetes API server where the entities are discovered).
+
+Commonly, a number of attributes of an entity are readily
+available for the telemetry producer to compose an identity. Of
+the available attributes the entity ID should include the
+minimal set of attributes that is sufficient for uniquely identifying that entity. For example a process on a host can be
+uniquely identified by (`process.pid`,`process.start_time`)
```

Comment:
Is this supposed to be [`process.creation.time`](https://github.com/open-telemetry/semantic-conventions/blob/main/model/process/registry.yaml#L191-L196)? That's not in the entities.yaml for process but it is in the registry.